xlinectl/src/command/lock.rs

```diff
@@ -0,0 +1,60 @@
+use clap::{arg, ArgMatches, Command};
+use tokio::signal;
+use xline_client::{
+    error::Result,
+    types::lock::{LockRequest, UnlockRequest},
+    Client,
+};
+
+use crate::utils::printer::Printer;
+
+/// Definition of `lock` command
+pub(crate) fn command() -> Command {
+    Command::new("lock")
+        .about("Acquire a lock, which will return a unique key that exists so long as the lock is held")
+        .arg(arg!(<lockname> "name of the lock"))
+}
+
+/// Build request from matches
+pub(crate) fn build_request(matches: &ArgMatches) -> LockRequest {
+    let name = matches.get_one::<String>("lockname").expect("required");
+    LockRequest::new().with_name(name.as_bytes())
```

Comment:
Shell we change this to `LockRequest::new(name.as_bytes())`

Reply:
done